Get-help.md

```diff
@@ -48,13 +43,45 @@ If you have a question about a pull request or issue, you can also reach out to
+
+You can also email your mentor with any questions. If you don't have a mentor, complete the steps on the [[wiki page for contributing code to Oppia|Contributing-code-to-Oppia]] and you'll be assigned one.
+
 ## How to Ask Good Questions
 
-At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. 
+
+### General guidelines to follow 
+
+* __Be clear and concise__: Clearly articulate your question to avoid confusion and allow others to understand it easily.
+* __Provide context__: If applicable, provide relevant background information to help others understand your question and situation.
+* __Be respectful__: Maintain a respectful and courteous tone in your communication to foster a positive and inclusive community environment.
+* __Use proper formatting__: Organize your question using paragraphs, bullet points, and appropriate formatting to enhance readability.
+* __Proofread your message__: Take a moment to proofread your question to ensure clarity and accuracy before sending it.
+
+### Providing error information 
+
+When reporting an error, follow these steps to communicate the issue effectively:
+
+* __Start with the command__: Rather than saying "I have an error," provide the actual command you ran. Copy and paste the command in your email or chat message to give recipients the complete context.
+* __Include the specific error message__: Share the exact error message you encountered. This helps others quickly identify the potential cause and provide relevant solutions or suggestions.
+* __Share reproduction steps__: Include the specific steps that recipients should follow to reproduce the error you encountered. This allows them to investigate the issue more effectively.
+* __Explain your efforts and conclusions__: Explain what you've tried and what conclusions you've drawn, if any. (Providing a [debugging doc](https://github.com/oppia/oppia/wiki/Debugging-Docs) may help.)
+
+## Building your skills: Technical writing
+
+In addition to effective communication, developing strong technical writing skills can greatly benefit your contributions. We recommend exploring the technical writing courses available at https://developers.google.com/tech-writing/overview. These courses can help you improve your ability to write clear code comments, create technical documentation, and effectively communicate complex ideas.
+
+In addition to this, We've also divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
```

Comment:
This feels out of place, especially since it's under the technical writing heading. I don't think it's necessary since the division between setup and general questions is clear from the table of contents. I think you can just delete this paragraph all together.